src/components/FilterList.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import { firestore } from '../lib/firebase';
+import { useCollection } from 'react-firebase-hooks/firestore';
+
+let arrString = ['fruit', 'vegetables', 'water', 'legumes'];
+const FilterList = () => {
+  const [value, setValue] = useState('');
```

Comment:
Both of these `useStates` are another example of where a small change could drastically improve readability. We should ask ourselves, "what is `value` really?" and "does this help someone reading this understand what this component is doing?"

I think `filter` might be what we're looking for. And for line 9, where we have a `useState` tracking `results`, it would be nice to see that called something like `filteredItems` since `results` doesn't really tell me what this is.